equinix/services/fabricv4/docs/AccessPoint.md

```diff
@@ -39,7 +39,7 @@ print(AccessPoint.to_json())
 # convert the object into a dict
 access_point_dict = access_point_instance.to_dict()
 # create an instance of AccessPoint from a dict
-access_point_form_dict = access_point.from_dict(access_point_dict)
+access_point_from_dict = AccessPoint.from_dict(access_point_dict)
```

Comment:
These changes in the docs are a result of https://github.com/OpenAPITools/openapi-generator/pull/18359, which fixes invalid code samples in docs.